CONTRIBUTING.md

```diff
@@ -414,6 +414,34 @@ extract them to a `JSON` file or integrate translations back to Kibana. To know
 We cannot support accepting contributions to the translations from any source other than the translators we have engaged to do the work.
 We are still to develop a proper process to accept any contributed translations. We certainly appreciate that people care enough about the localization effort to want to help improve the quality. We aim to build out a more comprehensive localization process for the future and will notify you once contributions can be supported, but for the time being, we are not able to incorporate suggestions.
 
+### Syling with SASS
+
+When writing a new component, create a sibling SASS file of the same name and import directly into the JS/TS component file. Doing so ensures the styles are never separated or lost on import and allows for better modularization (smaller individual plugin asset footprint).
+
+Any JavaScript (or TypeScript) file that imports SASS (.scss) files will automatically build with the EUI & Kibana invisibles (SASS variables, mixins, functions) from the `styling_constants.scss` file. However, any Legacy (file path includes `/legacy`) files will not.
```

Comment:
Since you're writing this up. Might want to include a simple one-liner to describe how this effects theming. Don't know how detailed we want to go here. Could probably be a paragraph about where the theme files live and how it gets toggled in the code.

Reply:
What do you mean specifically about "effects theming"? How it will inherently work in all themes (light and dark)? There aren't multiple theme files in Kibana. It does the dark to light mode switch in the build by replacing`eui_colors_light` with `eui_colors_dark`. I don't mind adding more, just trying to figure out what you need 😸

Reply:
I'm inclined to leave as-is, as really there's both no change from legacy to NP when it comes to theme switching and no change if they use EUI color vars. So I'll merge on green unless there's something specific you need.